paf-mvp-core-js/src/express/node.ts

```diff
@@ -143,6 +149,15 @@ export class Node implements INode {
     const { endPointName } = this.getRequestConfig(req);
     //req.correlationId() will get correlation-id from request header or generate a new one if it does not exist
     this.logger.Info(`${endPointName} --correlation-id=${req.correlationId()} - START`);
+    //Push a new span
+    let currentSpans = res.locals.spans;
```

Comment:
I think this can be simplified:
```typescript
res.locals.spans ??= []
```
and then line 160 the `currentSpans` variable is not needed anymore:
```typescript
res.locals.spans.push(currentSpan)
```

Reply:
Done